android/build.gradle

```diff
@@ -192,3 +192,58 @@ dependencies {
         }
     }
 }
+
+def isScreensExampleApp() {
+    return project.hasProperty('isScreensExampleApp') && project.property('isScreensExampleApp')
```

Comment:
Also, doesn't `project.property` return a string? We might want to compare it to `"true"` or something.

Reply:
When I used project.property in project that do not have this I got an exception that property does not exist

Reply:
Yeah yeah, but my point is that `project.property('isScreensExampleApp')` does return a `String` (I think), thus no matter what string is present there, the whole expression will be resolved to true.